deps/chakrashim/src/v8signature.cc

```diff
@@ -53,7 +53,7 @@ bool Utils::CheckSignature(Local<FunctionTemplate> receiver,
   if (!matched) {
     const char txt[] = "Illegal invocation";
     JsValueRef msg, err;
-    if (JsPointerToStringUtf8(txt, _countof(txt) - 1, &msg) == JsNoError &&
+    if (JsCreateString(txt, _countof(txt) - 1, &msg) == JsNoError &&
```

Comment:
Here it is `_countof() - 1`... Please check the jslib sources. Should we use `_countof() - 1` there, or `sizeof() - 1` or `sizeof()` (currently)?